drivers/hud_apr/app/models/hud_apr/generators/shared/fy2020/base.rb

```diff
@@ -159,7 +161,7 @@ def needs_ce_assessments?
             drug_abuse_entry: [2, 3].include?(disabilities_at_entry.detect(&:substance?)&.DisabilityResponse),
             drug_abuse_exit: [2, 3].include?(disabilities_at_exit.detect(&:substance?)&.DisabilityResponse),
             drug_abuse_latest: [2, 3].include?(disabilities_latest.detect(&:substance?)&.DisabilityResponse),
-            enrollment_coc: enrollment.enrollment_coc_at_entry&.CoCCode,
+            enrollment_coc: enrollment.enrollment_coc,
```

Comment:
I don't think we want to change this.  The logic for HUD reports is well specified.  This changes that logic if we ever need to go back and see why an answer is what it is.

Reply:
Okay. Sounds like we also need to keep the enrollment_coc_at_entry association on enrollment. I had hoped to remove it as that would better identify places where we missed updating code to reflect 2024 changes.